apps/server/src/modules/tool/service/validation/external-tool-validation.service.spec.ts

```diff
@@ -1,7 +1,7 @@
-import { externalToolDOFactory } from '@shared/testing/factory/domainobject/external-tool.factory';
+import { externalToolDOFactory } from '@shared/testing/factory/domainobject/tool/external-tool.factory';
```

Comment:
```suggestion
import { externalToolDOFactory } from '@shared/testing/factory/domainobject/tool';
```

???

Reply:
We could not change the import here because it would lead to an indirect dependency cycle. 
We analysed the issue and we are not able to resolve it so easily.
Maybe you have an idea to solve the problem..